### PR TITLE
Add PostgresPrefs v2.4.2

### PR DESCRIPTION
**Pull request by bashu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Preference Pane for administering PostgreSQL on Mac OS X

### CODE REVIEW

**Comment by miccal** [comment]
You have forgotten to append `.rb` to the Cask file.

File: Casks/postgresprefs.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'postgresprefs' do
+  version '2.4.2'
+  sha256 'fb3f618321efec12cbad21ec2d77e6a024a35f7a17bb304b421970f85e38337f'
+
+  url "https://github.com/MaccaTech/PostgresPrefs/releases/download/v#{version}/PostgreSQL.prefPane-v#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/postgresprefs.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/postgrespreferencepane.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'postgrespreferencepane' do
+  version '2.4.2'
+  sha256 'fb3f618321efec12cbad21ec2d77e6a024a35f7a17bb304b421970f85e38337f'
+
+  url "https://github.com/MaccaTech/PostgresPrefs/releases/download/v#{version}/PostgreSQL.prefPane-v#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/postgrespreferencepane.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/postgrespreferencepane.rb (hunk 2 above)
Comment (bashu) [inline Casks/postgrespreferencepane.rb]: @vitorgalvao done

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**